jina/__init__.py

```diff
@@ -55,6 +60,7 @@ def _warning_on_one_line(message, category, filename, lineno, *args, **kwargs):
 
     _set_start_method('fork')
```

Comment:
u have to do the same here

Reply:
I tested the case again and found it is not necessary to apply the try-catch here.

Reply:
because u removed the `pop` of the env variable.

Reply:
I think the macos patch above takes effect only when `JINA_MP_START_METHOD` is not set. Hence, try-catch does not make sense (the exception should be raised in this case).